tests/test_events.py

```diff
@@ -17,52 +17,108 @@
 
 
 @pytest.mark.asyncio
-# Not implemented yet.
-async def _ignore_test_consoleLog_logEntryAddedEventEmitted(websocket,
-      context_id):
+async def test_consoleLog_logEntryAddedEventEmitted(websocket):
+    context_id = await get_open_context_id(websocket)
```

Comment:
There is a fixture `context_id`, which provides an open context ID and can be used instead of getting it manually:
```suggestion
async def test_consoleLog_logEntryAddedEventEmitted(websocket, context_id):
```

Reply:
used context_id in the test cases.